src/App.js

```diff
@@ -22,33 +22,38 @@ const App = () => {
   const [hand, setHand] = useState([])
 
   const drawCard = () => {
-    // 1)
+    // 1) This variable holds a randomized value from the arrays suit and rank, it will randomly select an index with the Math function. 
```

Comment:
It won't randomly select an index, it is randomly selecting a value from each array.